infra/terraform/modules/kops_spec/main.tf

```diff
@@ -108,7 +108,7 @@ data "template_file" "bastions_instancegroup" {
     machine_type              = "${var.bastions_machine_type}"
     root_volume_size          = "${var.bastions_root_volume_size}"
     max_size                  = "${var.bastions_instancegroup_max_size}"
-    min_size                  = "${var.bastions_instancegroup_max_size}"
```

Comment:
🤨 what now

Reply:
The min size of the IG was set to equal the value of the max_size variable, we don't want that we want it to be the min_size variable

Reply:
Min size was set to be the value of the max_size variable, instead of the min_size variable.

Reply:
Yeah I was just wondering why the heck it was set as that!